brightwind/load/station.py

```diff
@@ -115,15 +115,18 @@ def _rename_to_title(list_or_dict, schema):
             prefixed_name = key + PREFIX_DICT[key]['prefix_separator'] + col
             prefixed_names[prefixed_name] = {'prefix_separator': PREFIX_DICT[key]['prefix_separator'],
                                              'title_prefix': PREFIX_DICT[key]['title_prefix']}
+    list_special_cases_no_prefix = ['sensor_config.slope', 'sensor_config.offset', 'sensor_config.sensitivity',
```

Comment:
Do we need to keep the 'sensor_config' ones? Just support from v1.0 onwards.

Reply:
solved